cmd/kubeadm/app/constants/constants.go

```diff
@@ -208,8 +209,33 @@ var (
 
 	// MinimumKubeletVersion specifies the minimum version of kubelet which kubeadm supports
 	MinimumKubeletVersion = version.MustParseSemantic("v1.8.0")
+
+	// SupportedEtcdVersion lists officially supported etcd versions with corresponding kubernetes releases
+	SupportedEtcdVersion = map[string]string{
+		"v1.7": "3.0.17",
```

Comment:
We don't support v1.7 anymore at HEAD so this is not necessary

Reply:
will remove it